### ISSUE TITLE
Discord status box in profile view not themed

### ISSUE THREAD

**Issue by ItsLiyua** [refact0r/system24#16]
The little box with the current status of the user is not themed as of right now. Other than that it's a great theme! Looking forward to using it.
![2024-09-08_14_42_43](https://github.com/user-attachments/assets/b67aa823-65d2-4a6e-86e5-7013005efcf8)

**Comment by ItsLiyua** [comment]
it looks a little different right now because I have enabled transparency for my discord window which is applied by Hyprland. the gray background above that is just me setting the default discord color.

**Comment by DaviDeMeo** [comment]
yeah, i fixed it by replacing https://github.com/refact0r/system24/blob/210dfa073fd216b5c7366243d52ee21816167860/src/unrounding.css#L29-L32
with `inset:75% -12.5% -12.5% 75%;`, it keeps the correct aspect ratio everywhere.

**Comment by TeapotExpress** [comment]
I found the solution!
In discord's accessibility settings change saturation to 100%
